andela/ah-baqi-frontend
# #166980211 Add CRUD functionality

### What does this PR do?
Enables creation, reading, updating, and deleting of articles.

### Description of Task to be completed?
- Checkout to:

      ft-crud-articles-166980211
     
- Install dependencies:

      yarn
     
- Run local server:

      yarn develop
     
### How should this be manually tested?
- Load home page to see all articles
- Click on a single article
- If you're the owner of the article, you'll be able to see editing and deleting options
- Click on `New Article` on the navbar to create a new article
### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[#166980211](https://www.pivotaltracker.com/story/show/166980211)
### Screenshots
<img width="739" alt="Screenshot 2019-07-17 at 20 20 09" src="https://user-images.githubusercontent.com/19375569/61396408-5852dc00-a8d0-11e9-8cf2-9cedadb79eb0.png">
